dolphinscheduler-alert/dolphinscheduler-alert-server/src/main/java/org/apache/dolphinscheduler/alert/AlertServerMetrics.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.dolphinscheduler.alert;
+
+import java.util.function.Supplier;
+
+import io.micrometer.core.instrument.Counter;
+import io.micrometer.core.instrument.Gauge;
+import io.micrometer.core.instrument.Metrics;
+
+public final class AlertServerMetrics {
+
+    private AlertServerMetrics() {
+        throw new UnsupportedOperationException("Utility class");
```

Comment:
Use `@UtilityClass` annotation in this class to reduce these boilerplate codes

Reply:
Sure, thx for the suggestion. Will update it.

Reply:
Besides constructors, it looks like `@UtilityClass` automatically marks all fields and methods in the class as `static final`. Should we remove all of them during refactoring? Or keep those `static final` along with `@UtilityClass` for readability?
https://projectlombok.org/features/experimental/UtilityClass

Reply:
I prefer to remove them. Utility class annotation takes care of all the conventions of a utility class